### DISCUSSION TITLE
Layer Diffusion in ComfyUI

### DISCUSSION THREAD

**Discussion by kakachiex2** [comfyanonymous/ComfyUI#2943] [General]
Is there any way to setup layer diffusion in comfyUI I'm experimenting with the lora but I think is not supported yet.
Here is the WebUI Forge implementation
https://github.com/layerdiffusion/sd-forge-layerdiffusion
https://huggingface.co/LayerDiffusion/layerdiffusion-v1/tree/main

**Comment by ynie** [comment]
+ 1

**Comment by camoody1** [comment]
+1

**Comment by melMass** [comment]
https://github.com/huchenlei/ComfyUI-layerdiffusion
